hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/SingleTransactionRecord.java

```diff
@@ -30,10 +32,14 @@
 public record SingleTransactionRecord(
         @NonNull Transaction transaction,
         @NonNull TransactionRecord transactionRecord,
-        @NonNull List<TransactionSidecarRecord> transactionSidecarRecords) {
+        @NonNull List<TransactionSidecarRecord> transactionSidecarRecords,
+        @NonNull TransactionOutputs transactionOutputs) {
     public SingleTransactionRecord {
         requireNonNull(transaction, "transaction must not be null");
         requireNonNull(transactionRecord, "record must not be null");
         requireNonNull(transactionSidecarRecords, "transactionSidecarRecords must not be null");
+        requireNonNull(transactionOutputs, "transactionOutputs must not be null");
     }
+
+    public record TransactionOutputs(@Nullable TokenType tokenType) {}
```

Comment:
We could probably add a comment this is used only by BlockStreams

Reply:
Done.